packages/components/src/menu-items-choice/types.ts

```diff
@@ -38,6 +38,10 @@ export type MenuItemChoice = {
 	 * Unique value for choice.
 	 */
 	value: string;
+	/**
+	 * Unique value for choice.
```

Comment:
Prop description needs an update. Perhaps something like:

```suggestion
	 * Whether the menu item is disabled.
```